lib/addons/kube-proxy/index.ts

```diff
@@ -1,15 +1,54 @@
+import { CfnAddon, KubernetesVersion } from "aws-cdk-lib/aws-eks";
+import { Construct } from "constructs";
+import { ClusterInfo } from "../../spi";
+import { logger } from "../../utils";
 import { CoreAddOn } from "../core-addon";
 
+const versionMap: Map<KubernetesVersion, string> = new Map([
+    [KubernetesVersion.of("1.27"), "v1.27.1-eksbuild.1"],
```

Comment:
Why is this line different?

Reply:
The CDK team hasn't added Kubernetes version 1.27 to their enum list yet